projects/packages/jetpack-mu-wpcom/src/features/wpcom-dashboard-widgets/wpcom-dashboard-widgets.php

```diff
@@ -49,7 +50,7 @@ function enqueue_wpcom_dashboard_widgets() {
 	$data = wp_json_encode(
 		array(
 			'siteName'    => wp_specialchars_decode( get_bloginfo( 'name' ), ENT_QUOTES ),
-			'siteDomain'  => wp_parse_url( home_url(), PHP_URL_HOST ),
+			'siteUrl'     => home_url(),
```

Comment:
What's the difference between the two things?

Reply:
One includes the protocol, the other doesn't. I need the full url to be able to create a link, and I'm using `new URL` in JS to derive the domain.